### PR TITLE
SyntaxException, FuncCallNode, FuncParamsNode, & TypeNode

### PR DESCRIPTION
**Pull request by uvu1688**

### CODE REVIEW

**Comment by trl6895** [comment]
Everything looks fine to me. Changes to `ProgramNode` and `JottParser` should be overrided by my PR but that's not really an issue

**Review by trl6895 — APPROVED** [review]
Looks good to me.  See comment

File: nodes/TypeNode.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            };
+        }
+        else {
+            throw new SyntaxException(tokens.get(0).getToken(), tokens.get(0).getFilename(), tokens.get(0).getLineNum());
+        }
+    }
+
+    @Override
+    public String convertToJott() {
+        return typeName;
```
Comment (m-dorsey) [inline nodes/TypeNode.java]: very very minor, `this.typeName`

**Review by m-dorsey — COMMENTED** [review]
Just a small addition to a missing `this.` Great work and pls merge :)

**Review by m-dorsey — APPROVED** [review]
Ty Uzo <3

**Pull request merged**